checks/raw/pinned_dependencies_test.go

```diff
@@ -449,7 +449,7 @@ func TestFileIsInVendorDir(t *testing.T) {
 		},
 		{
 			name:     "misspelled vendor dir",
-			filename: "a/vendorr/Dockerfile",
+			filename: "a/vendor_/Dockerfile",
```

Comment:
I understand that the goal of this test is to check for a misspelled item. I think that this misspelling should be functionally equivalent for validating the feature while making it easier to catch other misspellings in this project.

-- That said, I'm happy to drop this or anything else.